### PR TITLE
Update blacklist.json

### PR DESCRIPTION
**Pull request by Alawork**

### CODE REVIEW

**Comment by Alawork** [comment]
big list from [link](https://github.com/Discord-AntiScam/scam-links)

**Comment by Be1zebub** [comment]
why did you delete these domains? wtf
```
donkeytheme.taplink.ws
discord.gift
disorde.gift
discorl.gift
discofd.xyz
```

File: blacklist.json [hunk 1]
```diff
@@ -1,36 +1,6492 @@
 [
-	"discofd.xyz",
```
Comment (Be1zebub) [inline blacklist.json:2]:
```suggestion
	"discofd.xyz",
```

**Comment by Alawork** [comment]
изображение

I'll add the rest now, accidentally deleted

File: blacklist.json (hunk 1 above)
Comment (Alawork) [reply blacklist.json:2]: ?

**Comment by Be1zebub** [comment]
okay, I scanned the links you added, most of them contain phishing or are banned by the RKN.
I hope there are no good sites among them.

**Pull request merged**